docs/scos/dev/glue-api-guides/202304.0/dynamic-data-exchange-api/how-to-guides/how-to-send-request-in-dynamic-data-exchange-api.md

```diff
@@ -0,0 +1,457 @@
+
+Let's configure `isCreatable: false` for `iso3_code` and send the same request.
+You will receive the following response which does not include `iso3_code` because the value was not set:
+
+```json
+[
+    {
+        "iso2_code": "WA",
+        "name": "FOO",
+        "id_country": 259
+    }
+]
+```
+
+{% endinfo_block %}
+
+{% info_block infoBox %}
+
+It is important to consider that certain database-specific configurations may result in issues independent of entity configurations.
+
+For instance, in the case of MariaDB, it is not possible to set the ID value for an auto-incremented field. 
+Additionally, for the `iso2_code` field in the `spy_country` table, it must have a unique value. 
+Therefore, before creating a new record, it is necessary to verify that you do not pass a duplicated value for this field. 
+Otherwise, you will receive the following response:
+
+```json
+[
+    {
+        "message": "Failed to persist the data. Please verify the provided data and try again.",
+        "status": 500,
+        "code": "502"
+    }
+]
+```
+
+This response is caused by a caught Propel exception, specifically an integrity constraint violation `(Integrity constraint violation: 1062 Duplicate entry 'WA' for key 'spy_country-iso2_code')`. 
+
+{% endinfo_block %}
+
+### Sending `PATCH` request
+
+To update a collection of countries, submit the following HTTP request:
+
+```bash
+PATCH /dynamic-entity/country HTTP/1.1
+Host: glue-backend.mysprykershop.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer {your_token}
+Content-Length: 174
+{
+    "data": [
+        {
+            "id_country": 1,
+            "iso2_code": "WB",
+            "iso3_code": "WWB",
+            "name": "FOO"
+        }
+    ]
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+If everything is set up correctly you will get the following response:
+
+```json
+[
+  {
+    "iso2_code": "WB",
+    "iso3_code": "WWB",
+    "name": "FOO",
+    "id_country": 1
+  }
+]
+```
+
+The response payload includes all the specified fields from the request body.
+
+{% endinfo_block %}
+
+{% info_block infoBox %}
+
+It is also possible to send a `PATCH` request for a specific ID instead of a collection, use the following request:
+
+```bash
+PATCH /dynamic-entity/country/1 HTTP/1.1
+Host: glue-backend.mysprykershop.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer {your_token}
+Content-Length: 143
+{
+    "data": {
+            "iso2_code": "WB",
+            "iso3_code": "WWB",
+            "name": "FOO"
+        }
+}
+```
+
+{% endinfo_block %}
+
+{% info_block infoBox %}
+
+Note that all fields included in the request payload are marked as `isEditable: true` in the configuration.
+Therefore, the API will update the found record with all the provided fields.
+However, if any of the fields are configured as `isEditable: false` it won't be updated.
```

Comment:
If any of those fields are not editable, but we are trying to edit them, we should receive 400 response with a descriptive message.
Please verify the implementation, and update the documentation.